eq-author/src/App/QuestionnaireDesignPage/MainNavigation/index.js

```diff
@@ -88,6 +89,7 @@ export const UnwrappedMainNavigation = ({
   useSubscription(publishStatusSubscription, {
     variables: { id: params.questionnaireId },
   });
+  const totalErrorCountNoTheme = totalErrorCount - themeErrorCount;
```

Comment:
I'm not sure if we should make this a little more specific. 

If the user has turned off all themes there won't be a preview theme selected and therefore the View Survey button won't, or shouldn't, work.

But, if they haven't entered something like the form type or survey ID, then they should still be able to view.

I don't think this was in the ACs though, so it's probably something we can add on another iteration. 

What do you think?

Reply:
The ticket’s AC included allowing view survey to be clicked when the form type is invalid. I think the name themeErrorCount makes it misleading as you mentioned in another comment, that value represents only the form type validation error. This code is allowing view survey to be clicked when there is a form type error, and prevents it from being clicked when there is any other error, including if no theme is enabled.